src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.ts

```diff
@@ -207,27 +208,43 @@ export class NfsFormComponent extends CdForm implements OnInit {
 
   setSubVolGrpPath(): Promise<void> {
     return new Promise<void>((resolve, reject) => {
-      const subvolGroup = this.nfsForm.getValue('subvolume_group');
-      const fs_name = this.nfsForm.getValue('fsal').fs_name;
-
-      if (subvolGroup === this.defaultSubVolGroup) {
-        this.updatePath('/volumes/' + this.defaultSubVolGroup);
-        resolve();
-      } else {
+      const { fsName, subvolGroup } = this.checkVolumeValidation();
+      if (subvolGroup) {
         this.subvolgrpService
-          .info(fs_name, subvolGroup)
-          .pipe(map((data) => data['path']))
-          .subscribe(
-            (path) => {
-              this.updatePath(path);
-              resolve();
-            },
-            (error) => reject(error)
-          );
+        .info(fsName, subvolGroup)
+        .pipe(map((data) => data['path']))
+        .subscribe(
+          (path) => {
+            this.updatePath(path);
+            resolve();
+          },
+          (error) => reject(error)
+        );
+      } else {
+        this.updatePath('');
+        this.setUpVolumeValidation();
```

Comment:
iN checkVolumeValidation we are setting path and validation and here again, is it not redoing/overwriting ?